frontend/src/utils/customHooks/useDisableElementsOnSomeRoutes.js

```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from 'react'
+import { useLocation } from 'react-router-dom'
+
+export const useDisableElementsOnSomeRoutes = props => {
+  const location = useLocation()
+  const [disable, setDisable] = useState(false)
+
+  useEffect(() => {
+    if (props) {
+      props.map(path => {
+        return location.pathname.includes(path)
+          ? setDisable(true)
+          : setDisable(false)
+      })
```

Comment:
This hook only works properly with only one route provided. 
When there is two or more, it will not work as expected.
Try yourself in the browser ;)

Reply:
Okey :)